types/sparkly/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for sparkly 3.1
+// Project: https://github.com/sindresorhus/sparkly
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = sparkly;
+
+declare function sparkly(numbers: any[], options?: sparkly.Options): string;
```

Comment:
Technically, passing `NaN` or `Infinity` would allow to specify holes, too. But wouldn't it be awkward to create typings that don't work with the examples on the project page?